clientd3d/statgame.c

```diff
@@ -197,16 +206,78 @@ void GameMouseButtonDown(HWND hwnd, BOOL fDoubleClick, int x, int y, UINT keyFla
    UserDidSomething();   /* User is alive! */
 
    SetFocus(hMain);
+
+   if (text_area_resize_zone) {
+       // Start text area resize.
+       text_area_resize_start_x = x;
+       text_area_resize_start_y = y;
+       text_area_resize_inprogress = true;
+   }
+
 }
 /****************************************************************************/
 void GameLButtonUp(HWND hwnd, int x, int y, UINT keyFlags)
 {
-   UserEndDrag();
+    if (text_area_resize_inprogress) {
+        // End text area resize.
+        text_area_resize_zone = false;
+        text_area_resize_inprogress = false;
+    }
+
+    UserEndDrag();
 }
 /****************************************************************************/
 void GameMouseMove(HWND hwnd, int x, int y, UINT keyFlags)
 {
-   //GameWindowSetCursor();
+    RECT rcToolbar;
+    ToolbarGetUnionRect(&rcToolbar);
+
+    /* Resize the text area is possible from the left hand edge of the client window,
+    in-between the viewport and textarea. The vertical resize cursor will be displayed
+    at this time. */
+    int offset = 5;
+    if (text_area_resize_inprogress ||
+        (y >= main_viewport_height + rcToolbar.bottom + EDGETREAT_HEIGHT- offset
+        && y <= main_viewport_height + EDGETREAT_HEIGHT + rcToolbar.bottom + offset
+        && x <= EDGETREAT_HEIGHT + offset))
```

Comment:
Can we have the right edge of this window extend to the right edge of the text area?  That will be a lot more discoverable.